<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LarvaTab</class>
 <widget class="QWidget" name="LarvaTab">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>374</width>
    <height>532</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>350</width>
       <height>485</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>350</width>
       <height>485</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Helvetica</family>
      </font>
     </property>
     <property name="title">
      <string/>
     </property>
     <property name="checkable">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QWidget" name="widgetChanged" native="true">
        <property name="font">
         <font>
          <family>Helvetica</family>
         </font>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>9</number>
         </property>
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="font">
            <font>
             <family>Helvetica</family>
            </font>
           </property>
           <property name="title">
            <string>Larvaposition has been changed manualy</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
           <property name="flat">
            <bool>false</bool>
           </property>
           <property name="checkable">
            <bool>false</bool>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_8">
            <item>
             <widget class="QPushButton" name="pushButtonApplyChanges">
              <property name="font">
               <font>
                <family>Helvetica</family>
               </font>
              </property>
              <property name="text">
               <string>Apply changes</string>
              </property>
              <property name="icon">
               <iconset resource="Icons.qrc">
                <normaloff>:/ResultViewer/Icons/Icons/rect2985.png</normaloff>:/ResultViewer/Icons/Icons/rect2985.png</iconset>
              </property>
              <property name="flat">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButtonReset">
              <property name="font">
               <font>
                <family>Helvetica</family>
               </font>
              </property>
              <property name="text">
               <string>Reset</string>
              </property>
              <property name="icon">
               <iconset resource="Icons.qrc">
                <normaloff>:/ResultViewer/Icons/Icons/exit.png</normaloff>:/ResultViewer/Icons/Icons/exit.png</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="pushButtonColorDialog">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Icons.qrc">
            <normaloff>:/ResultViewer/Icons/Icons/color.png</normaloff>:/ResultViewer/Icons/Icons/color.png</iconset>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="labelColorDialog">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="autoFillBackground">
           <bool>true</bool>
          </property>
          <property name="frameShape">
           <enum>QFrame::Box</enum>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QCheckBox" name="checkBoxDisplayLarva">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>Show Larva</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="pushFindLarva">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="toolTip">
           <string>Szene zu dieser Larve scrollen</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="Icons.qrc">
            <normaloff>:/ResultViewer/Icons/Icons/pathfinder.png</normaloff>:/ResultViewer/Icons/Icons/pathfinder.png</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="3" column="2">
         <widget class="QCheckBox" name="checkBoxPath">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>Show</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label_5">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>Distance traveled:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QPushButton" name="pushButtonPrevCollision">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Helvetica</family>
             </font>
            </property>
            <property name="text">
             <string>&lt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButtonNextCollision">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Helvetica</family>
             </font>
            </property>
            <property name="text">
             <string>&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="1">
         <widget class="QLabel" name="labelInterpolated">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>Ambiguity Eventl:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>Distance to origin:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="labelDist2Origin">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QCheckBox" name="checkBoxDist2Origin">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>Show</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLabel" name="labelAccDistance">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="labelFrameView">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>0/0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderTime">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxTime">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="Line" name="line">
        <property name="font">
         <font>
          <family>Helvetica</family>
         </font>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <property name="topMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>Radius at point</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinBoxRadiusPosition">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>:</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="doubleSpinBoxRadius">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>0</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="suffix">
           <string> Pixel</string>
          </property>
          <property name="decimals">
           <number>2</number>
          </property>
          <property name="singleStep">
           <double>0.500000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushInvert">
        <property name="font">
         <font>
          <family>Helvetica</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">Text-align:left</string>
        </property>
        <property name="text">
         <string>Invert Model</string>
        </property>
        <property name="icon">
         <iconset resource="Icons.qrc">
          <normaloff>:/ResultViewer/Icons/Icons/invert.png</normaloff>:/ResultViewer/Icons/Icons/invert.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushAttach">
        <property name="font">
         <font>
          <family>Helvetica</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">Text-align:left</string>
        </property>
        <property name="text">
         <string>Attach Another Model</string>
        </property>
        <property name="icon">
         <iconset resource="Icons.qrc">
          <normaloff>:/ResultViewer/Icons/Icons/unite.png</normaloff>:/ResultViewer/Icons/Icons/unite.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_12">
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>Insert Timestep Duplicate:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushCopy2Prev">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>before</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushCopy2Succ">
          <property name="font">
           <font>
            <family>Helvetica</family>
           </font>
          </property>
          <property name="text">
           <string>after</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="pushDeleteCurrent">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="font">
         <font>
          <family>Helvetica</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">Text-align:left</string>
        </property>
        <property name="text">
         <string>Delete Model In The Current Frame</string>
        </property>
        <property name="icon">
         <iconset resource="Icons.qrc">
          <normaloff>:/ResultViewer/Icons/Icons/exit.png</normaloff>:/ResultViewer/Icons/Icons/exit.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushDeleteLarva">
        <property name="palette">
         <palette>
          <active>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </active>
          <inactive>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>0</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>255</red>
              <green>0</green>
              <blue>0</blue>
             </color>
            </brush>
           </colorrole>
          </inactive>
          <disabled>
           <colorrole role="Text">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
           <colorrole role="ButtonText">
            <brush brushstyle="SolidPattern">
             <color alpha="255">
              <red>120</red>
              <green>120</green>
              <blue>120</blue>
             </color>
            </brush>
           </colorrole>
          </disabled>
         </palette>
        </property>
        <property name="font">
         <font>
          <family>Helvetica</family>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">Text-align:left</string>
        </property>
        <property name="text">
         <string>Remove Larva From Results</string>
        </property>
        <property name="icon">
         <iconset resource="Icons.qrc">
          <normaloff>:/ResultViewer/Icons/Icons/exit.png</normaloff>:/ResultViewer/Icons/Icons/exit.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="font">
      <font>
       <family>Helvetica</family>
      </font>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Icons.qrc"/>
 </resources>
 <connections/>
</ui>
